tox.ini

```diff
@@ -49,6 +49,7 @@ passenv =
 deps =
     -rrequirements.txt
     factory-boy
+    faker<=5.4.0
```

Comment:
Issue in https://github.com/joke2k/faker/issues/1367

Limiting until they make a release to resolve this